evennia/objects/objects.py

```diff
@@ -311,22 +319,22 @@ def exits(self):
     # main methods
 
     def search(
-        self,
-        searchdata,
-        global_search=False,
-        use_nicks=True,
-        typeclass=None,
-        location=None,
-        attribute_name=None,
-        quiet=False,
-        exact=False,
-        candidates=None,
-        use_locks=True,
-        nofound_string=None,
-        multimatch_string=None,
-        use_dbref=None,
-        tags=None,
-        stacked=0,
+            self,
+            searchdata,
+            global_search=False,
+            use_nicks=True,
+            typeclass=None,
+            location=None,
+            attribute_name=None,
+            quiet=False,
+            exact=False,
+            candidates=None,
+            use_locks=True,
+            nofound_string=None,
+            multimatch_string=None,
+            use_dbref=None,
+            tags=None,
+            stacked=0,
```

Comment:
Not sure which auto-formatter you use here, but 8 character indent seems to not be `black` standard.